Makefile

```diff
@@ -0,0 +1,38 @@
+
```

Comment:
Actually, having the binaries checked in the repo creates _binary noise_. What about ignoring them and instructing the build system to build only the binary for the system being built on? We can always keep the rules for individual platforms, so as to be able to build them on demand (for packaging purposes, for example).